msgpack-core/src/main/java/org/msgpack/core/buffer/ByteBufferInput.java

```diff
@@ -64,4 +64,16 @@ public void close()
     {
         // Nothing to do
     }
+
+    /**
+     * Create a ByteBufferInput on off heap memory
+     * @param address the address
+     * @param offset the offset
+     * @param length the length
+     * @return a new ByteBufferInput on the specified address
+     */
+    public static ByteBuffer directBuffer(long address, int offset, int length)
+    {
+        return DirectBufferAccess.newByteBuffer(address, offset, length, null);
```

Comment:
It doesn't seem to me that this method is related to ByteBufferInput.

@xerial Do you think this is the right place to put the method in?

Reply:
We can make DirectBufferAccess a public class instead of adding this method.

Reply:
Actually this location is a bit weird.